modules/oadp-backup-restore-for-large-usage.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+// about-installing-oadp.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-backup-restore-for-large-usage_{context}"]
+= NodeAgent CPU for large usage
+
+Testing shows that increasing `NodeAgent` CPU can significantly improve backup and restore times when using {oadp}.
```

Comment:
```suggestion
Testing shows that increasing `NodeAgent` CPU can significantly improve backup and restore times when using {oadp-short}.
```

Reply:
@kaovilai Since it is the first usage in the module, it actually should be {oadp-first}, but thanks for keeping me honest. :)